src/components/Summary/index.js

```diff
@@ -120,6 +116,20 @@ const Summary = ({
   const onSubmit = async event => {
     event.preventDefault();
     if (refreshedSubmission.submissionAllowed !== SUBMISSION_ALLOWED.yes) return;
+
+    // Update the submission to persist that the privacy policy was accepted
+    try {
+      const response = await patch(submission.url, submission);
```

Comment:
This now persists the `privacy_policy_accepted` attribute when the user clicks on 'submit'. It could also happen when the user checks the box, but then we need to block the submit button while that request is happening, make sure that the submission is refreshed in the state, maybe deal with debouncing in case the user clicks many times... So i went for the other option :shrug: